### ISSUE TITLE
pub should download packages over https

### ISSUE THREAD

**Issue by DartBot** [dart-lang/pub#118]
<a href="https://github.com/jmesserly"><img src="https://avatars.githubusercontent.com/u/1081711?v=3" align="left" width="96" height="96"hspace="10"></img></a> **Issue by [jmesserly](https://github.com/jmesserly)**
_Originally opened as dart-lang/sdk#5191_

----

For what it's worth, the website appears to support HTTPS already:
https://pub.dartlang.org

So I think this is blocked on dart:io support (see &quot;blocked on&quot;)

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

*Added this to the [Later](../milestones/Later) milestone.*

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

*Removed [Type-Defect](../labels/Type-Defect) label.*
*Added [Type-Enhancement](../labels/Type-Enhancement) label.*

**Comment by DartBot** [comment]
<a href="https://github.com/dgrove"><img src="https://avatars.githubusercontent.com/u/2108507?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [dgrove](https://github.com/dgrove)**

----

*Removed this from the [Later](../milestones/Later) milestone.*
*Added this to the [M3](../milestones/M3) milestone.*

**Comment by DartBot** [comment]
<a href="https://github.com/nex3"><img src="https://avatars.githubusercontent.com/u/188?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [nex3](https://github.com/nex3)**

----

*Marked this as being blocked by dart-lang/sdk#7395.*

**Comment by DartBot** [comment]
<a href="https://github.com/nex3"><img src="https://avatars.githubusercontent.com/u/188?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [nex3](https://github.com/nex3)**

----

This will be fixed along with issue dart-lang/sdk#7395, so I'm merging it in.
______
*Added [Duplicate](../labels/Duplicate) label.*
*Marked as being merged into dart-lang/sdk#7395.*
